src/views/virtualmachines/actions/components/VirtualMachineMigration/tabs/VirtualMachineMigrationReviewTab.tsx

```diff
@@ -58,6 +60,12 @@ const VirtualMachineMigrationReviewTab: FC<VirtualMachineMigrationReviewTabProps
               </Td>
               <Td>{t('VirtualMachine storage')}</Td>
             </Tr>
+            <Tr>
+              <Td width={30}>
+                <strong>{t('Source StorageClasses')}</strong>
+              </Td>
+              <Td>{vmStorageClassNames.join(', ')}</Td>
```

Comment:
comma-delimited list is consistent with what we have here already but I would look for different solutions. How about using plain text if there is one item and an unordered list when there are more ?

Reply:
there is no space for unordered list. But in general the vm should have max 1 or 2 storageclasses so there is no issue with that. maybe i can discuss this with yifat and create an enhancement in a different pr